lib/broadside/configuration/deploy.rb

```diff
@@ -22,14 +22,18 @@ class DeployConfig < ConfigStruct
         :env_vars,
```

Comment:
you know that `Aws::Client` already provides stubs, right? https://docs.aws.amazon.com/sdkforruby/api/Aws/ClientStubs.html